internal/daysteps/daysteps.go

```diff
@@ -13,8 +20,42 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	newSlice := strings.Split(data, ",")
+	if len(newSlice) != 2 {
+		return 0, 0, errors.New("Неверное количество аргументов")
+	}
+	steps, err := strconv.Atoi(newSlice[0])
+	if err != nil {
+		return 0, 0, err
+	}
+
+	if steps <= 0 {
+		return 0, 0, errors.New("неверные шаги")
```

Comment:
Лучше точнее описывать суть проблемы
```suggestion
		return 0, 0, fmt.Errorf("steps must be positive")
```